dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/service/DrcLoggingIntegrationTest.java

```diff
@@ -201,8 +202,8 @@ void givenSomeActiveConcorContributionsAndProcessDailyFilesRan_whenDrcRespondsWi
         softly.assertThat(logged.getContributionFileIds()).containsExactly(contributionFileId, contributionFileId, contributionFileId); // 3.
         softly.assertThat(logged.getConcorContributionIds()).containsOnlyOnceElementsOf(updatedIds);
 
-        // TODO uncomment after fix 3 actual: softly.assertThat(contributionFile.getRecordsReceived()).isEqualTo(0); // 4.
-        softly.assertThat(contributionFile.getDateReceived()).isBeforeOrEqualTo(startDate);
+        softly.assertThat(contributionFile.getRecordsReceived()).isIn(0, null); // 4. (either zero or NULL)
```

Comment:
Is there a difference here we would want to look at, between null and 0?
As in, would null return be a different cause than 0?

Reply:
Right now, the code right returns NULL. But both NULL and 0 are equally valid return values meaning no "good" records received. Personally, I think the DB column should be `NOT NULL DEFAULT 0` rather than nullable, so I allowed for either value.